plugin/pkg/scheduler/metrics/metrics.go

```diff
@@ -68,3 +80,17 @@ func Register() {
 func SinceInMicroseconds(start time.Time) float64 {
 	return float64(time.Since(start).Nanoseconds() / time.Microsecond.Nanoseconds())
 }
+
+// ReportBindingRateLimiterSaturation exports status of binding rate limiter
+// whether it's full (1) or not (0).
+func ReportBindingRateLimiterSaturation(rl util.RateLimiter) {
+	for {
+		if rl.CanAccept() {
+			BindingRateLimiterSturation.Set(0)
```

Comment:
Seems like you can do a lot better by using 1 - (# tokens available) / (# tokens max). Then you could perhaps reduce the rate.

The way you have it now, suppose it's *always* used 19 out of 20 tokens. Then your metric will show 0, when really it should be .95. If code changes ever so slightly to use one more, it will instantly switch from 0 to 1.